### PR TITLE
update libg

### PR DESCRIPTION
**Pull request by pinzart90**
Update libg package references to include the latest linux binaries

### Purpose

(FILL ME IN) This section describes why this PR is here. Usually it would include a reference 
to the tracking task that it is part or all of the solution for.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/DynamoCore/DynamoCore.csproj [hunk 1]
```diff
@@ -29,11 +31,14 @@
...
     <PackageReference Include="Autodesk.IDSDK" Version="1.1.6" />
     <PackageReference Include="DynamoVisualProgramming.Analytics" Version="3.0.0.2506" />
-    <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.17.1.3967" GeneratePathProperty="true"/>
-    <PackageReference Include="DynamoVisualProgramming.LibG_228_0_0" Version="2.17.1.3967" GeneratePathProperty="true"/>
+    <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.18.0.646" GeneratePathProperty="true"/>
+    <PackageReference Include="DynamoVisualProgramming.LibG_228_0_0" Version="2.18.0.654" GeneratePathProperty="true"/>
     <PackageReference Include="Greg" Version="2.3.0.2505" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" CopyXML="true" />
 	  <PackageReference Include="RestSharp" Version="106.12.0" CopyXML="true" />
+    <Reference Include="System.Collections.Immutable">
```
Comment (mjkkirschner) [inline src/DynamoCore/DynamoCore.csproj:39]: what is this change for?

Comment (mjkkirschner) [reply src/DynamoCore/DynamoCore.csproj:39]: it looks quite strange - it's a reference to `System.Collections.Immutable` - but pointing at Protogeometry?

File: src/DynamoCore/DynamoCore.csproj [hunk 2]
```diff
@@ -122,14 +127,14 @@
   </Choose>
   <Target Name="BeforeBuildOps" BeforeTargets="Build">
     <PropertyGroup>
-      <LibGProtoGeometryLibGLocale>$(PkgDynamoVisualProgramming_LibG_229_0_0)\tools\netstandard2.0\libg_locale</LibGProtoGeometryLibGLocale>
+      <LibGProtoGeometryLibGLocale>$(PkgDynamoVisualProgramming_LibG_229_0_0)\tools\libg_locale</LibGProtoGeometryLibGLocale>
     </PropertyGroup>
     <ItemGroup>
-      <LibGInterface Include="$(PkgDynamoVisualProgramming_LibG_229_0_0)\tools\netstandard2.0\LibG.Interface.dll" />
-      <LibGProtoGeometryXml Include="$(PkgDynamoVisualProgramming_LibG_229_0_0)\lib\netstandard2.0\ProtoGeometry.XML" />
```
Comment (mjkkirschner) [inline src/DynamoCore/DynamoCore.csproj:129]: was the package layout modified?

**Review by mjkkirschner — CHANGES_REQUESTED** [review]
some questions

File: src/DynamoCore/DynamoCore.csproj (hunk 1 above)
Comment (pinzart90) [reply src/DynamoCore/DynamoCore.csproj]: That was supposed to be ProtoGeometry

File: src/DynamoCore/DynamoCore.csproj (hunk 2 above)
Comment (pinzart90) [reply src/DynamoCore/DynamoCore.csproj:129]: yes
I had to add both windows and linux flavors of all binaries (including managed)

File: src/DynamoCore/DynamoCore.csproj (hunk 1 above)
Comment (pinzart90) [reply src/DynamoCore/DynamoCore.csproj]: Ugh..I see now that indirect references are not turned on..

**Comment by mjkkirschner** [comment]
@pinzart90 something still seems wrong as all builds are failing.

File: src/DynamoCore/DynamoCore.csproj [hunk 3]
```diff
@@ -147,9 +152,12 @@
     <Copy SourceFiles="@(NodeHelpFiles)" DestinationFolder="$(OutputPath)\en-US\fallback_docs\" />
     <Copy SourceFiles="@(OpenSourceLicenses)" DestinationFolder="$(OutputPath)Open Source Licenses\" />
     <Copy SourceFiles="@(LibGInterface)" DestinationFolder="$(OutputPath)" />
+    <Copy SourceFiles="@(LibGProtoGeometryDLL)" DestinationFolder="$(OutputPath)" />
     <Copy SourceFiles="@(LibGProtoGeometryXml)" DestinationFolder="$(OutputPath)" />
     <Copy SourceFiles="@(LibGProtoGeometryXml)" DestinationFolder="$(OutputPath)\$(UICulture)" />
     <Copy SourceFiles="@(LibGProtoGeometryUICulture)" DestinationFolder="$(OutputPath)\$(UICulture)" />
+    <Copy Condition="$(Platform.Contains('Linux'))" SourceFiles="@(LibG229Deps)" DestinationFolder="$(OutputPath)libg_228_0_0\asm_deps\" />
```
Comment (mjkkirschner) [inline src/DynamoCore/DynamoCore.csproj:159]: @annabunches FYI

**Review by mjkkirschner — APPROVED** [review]

**Pull request merged**